templates/zerver/help/reply-to-message.md

```diff
@@ -0,0 +1,58 @@
+# Replying to a Message
+There are three ways to reply to a message in Zulip.
+## Clicking on a message
+To reply to a message, click on the message.
+A message compose box will appear with the **Stream** (outlined in red below)
+and **Topic** (outlined in green below) fields
+autofilled to match the message you clicked on.
+
+![reply messaging box](/static/images/help/reply-messaging-box.png)
+
+You can type your reply in the **Compose your message here...** field.
+Learn about formatting messages [here](/help/format-your-message-using-markdown).
+
+## The chevron icon
+Hover over the message that you want to reply to.
+Three icons will appear at the far right side of the message.
+
+![hover icons](/static/images/help/message-hover.png)
+
+Click on the chevron (![chevron](/static/images/help/chevron.png)) icon.
+This menu will appear:
+
+![menu](/static/images/help/menu-chevron-message.png)
+
+Click on **Reply**. A message compose box will appear with the **Stream** (outlined in red below)
+and **Topic** (outlined in green below) fields
+autofilled to match the message you selected.
+
+![reply messaging box](/static/images/help/reply-messaging-box.png)
+
+You can then type your reply in the **Compose your message here...** field.
+
+## The r hotkey
+On the Zulip home screen, you will notice a blue box around one of the
+messages on your screen.
+
+![blue box](/static/images/help/blue-box-message.png)
+
+This box indicates which message is currently highlighted.
+It is found on the home screen, in all streams, in your private
+messages, in your starred messages, and in your @-mentions.
+
+You can change which message is highlighted by scrolling or
+using the arrow keys on your keyboard.
+Pressing the `r` key on your keyboard will reply to whichever message is
+highlighted. A message compose box will appear with the **Stream**
+(outlined in red below)
+and **Topic** (outlined in green below) fields
+autofilled to match the message you selected.
+
+![reply messaging box](/static/images/help/reply-messaging-box.png)
+
+You can then type your reply in the **Compose your message here...**
+field.
+
+At any time you can discard your reply by clicking the **x**
```

Comment:
I think this one deserves it's own header and maybe a link to the documentation on drafts?

Also, `Esc` is closing the edit window too.